schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -461,6 +463,10 @@
         "product": {
           "type": "string",
           "description": "Name of the product"
+        },
```

Comment:
This is not for appliance. It's a general change. 

---
In reply to: [115818793](https://github.com/Azure/azure-resource-manager-schemas/pull/316#discussion_r115818793) [](ancestors = 115818793)